pages/phases/job-analysis/creating-assessments-from-competencies.md

```diff
@@ -37,6 +37,12 @@ If you are creating a new passing score work simulation assessment, think throug
 
 See a longer list at [this reference link](https://www.opm.gov/policy-data-oversight/assessment-and-selection/other-assessment-methods/).
 
+## Written Assessments Tips
+- The assessment type should be appropriate to evaluate the applicant's proficiency level within the examined competencies.
+- Ideally, you will have time during the workshop for live testing of the assessment questions. If you end up revising the assessment after the workshop, the revised assessment will need to be tested.
+- You should as part of testing record the time it will take to conduct the written assessment and added that to the job announcement.
```

Comment:
```suggestion
- You should as part of testing record the time it will take to conduct the written assessment and added that to the job announcement and to the assessment instructions.
```